terraform/environments/data-platform/tests/run_smoke_tests.py

```diff
@@ -34,53 +32,126 @@ def md5_hash_file_contents(file) -> str:
+        self.headers = {"authorizationToken": auth_token}
+        
+    def register(self):
+        print("Registering data product...")
+
+        metadata = {
+            "name": "example_prison_data_product",
+            "description": "just testing the metadata json validation/registration",
+            "domain": "MoJ",
+            "dataProductOwner": "matthew.laverty@justice.gov.uk",
+            "dataProductOwnerDisplayName": "matt laverty",
+            "email": "matthew.laverty@justice.gov.uk",
+            "status": "draft",
+            "dpiaRequired": False,
+            "retentionPeriod": 3650
+        }
+
+        return requests.post(
+            url=self.register_url,
+            headers=self.headers,
+            json={
+                "metadata": metadata
+            }
+        )
+
+    def delete_data_product(self):
+        print("Deleting data product...")
+        return requests.delete(
+            url=self.data_product_url,
+            headers=self.headers,
+        )
+
+    def create_schema(self):
+        print("Creating a schema...")
+        return requests.post(
+            url=self.table_url + "/schema",
+            headers=self.headers,
+            json={
+                "schema": {
+                    "tableDescription": "just a test table",
+                    "columns": [
+                        {"name": "col_0", "type": "double", "description": "just a test column"},
+                        {"name": "col_1", "type": "double", "description": "just a test column"},
+                        {"name": "col_2", "type": "double", "description": "just a test column"},
+                        {"name": "col_3", "type": "double", "description": "just a test column"},
+                    ]
+                }
+            }
+        )
+
+    def upload_file(self):
+        file_md5_hash = md5_hash_file_contents(filename)
+
+        body = {
+            "filename": filename,
+            "contentMD5": file_md5_hash,
+        }
+
+        # Get presigned url
+        response = requests.post(
+            url=self.presigned_url,
+            json=body,
+            headers=self.headers,
+        )
+
+        if response.status_code != 200:
+            print(f"Error getting presigned url. Status code: {response.status_code}")
+            print(f"Error getting presigned url. Response: {response.text}")
+            raise Exception('Error getting presigned URL')
+        
+        response_json = response.json()
+        post_policy_form_data = response_json["URL"]["fields"]
+        multipart_form_data = {
+            **post_policy_form_data,
+            "file": (post_policy_form_data["key"], open(filename, "r")),
+        }
+
+        # Upload data
+        print("Uploading data")
+        return requests.post(response_json["URL"]["url"], files=multipart_form_data)
+
+
+def run_test(client):
+    # register_response = client.register()
```

Comment:
Why this is commented ? can it not be removed ?

Reply:
I was hoping to add this back in when the delete endpoint is updated, but can remove if you prefer?